caas/kubernetes/provider/bootstrap.go

```diff
@@ -1374,14 +1422,14 @@ func (c *controllerStack) buildContainerSpecForController() (*core.PodSpec, erro
 		fmt.Sprintf("controller-%s", c.pcfg.ControllerId),
 		agentconstants.AgentConfigFilename,
 	)
-	var jujudCmds []string
-	pushCmd := func(cmd string) {
-		jujudCmds = append(jujudCmds, cmd)
-	}
+
+	jujudEnv := map[string]string(nil)
```

Comment:
As much as I prefer `jujudEnv := map[string]string(nil)`
`var jujudEnv map[string]string` is the common way across the project

Reply:
How about
```
var jujudEnv map[string]string = nil
```
nice to be explicit